### ISSUE TITLE
Instance crashes here and there. 

### ISSUE THREAD

**Issue by TMFuzz** [nikitavondel/steam-ts#18]
[npm-debug.log.txt](https://github.com/nikitavondel/steam-ts/files/936357/npm-debug.log.txt)

Admits having this deployed a created its own query login.
I have another bot accessing the querry. Via its own query login also.
I have no idea if this is relational but steam bot crashes after a day or so.

`/home/ubuntu/steam-ts/node_modules/async/dist/async.js:837
          if (fn === null) throw new Error("Callback was already called.");
                           ^

Error: Callback was already called.
    at /home/ubuntu/steam-ts/node_modules/async/dist/async.js:837:34
    at Object.cb (/home/ubuntu/steam-ts/lib/queryusers.js:43:29)
    at LineInputStream.<anonymous> (/home/ubuntu/steam-ts/node_modules/node-teamspeak/index.js:170:35)
    at emitOne (events.js:96:13)
    at LineInputStream.emit (events.js:188:7)
    at LineInputStream.line (/home/ubuntu/steam-ts/node_modules/node-teamspeak/node_modules/line-input-stream/lib/line-input-stream.js:8:8)
    at Array.forEach (native)
    at Socket.<anonymous> (/home/ubuntu/steam-ts/node_modules/node-teamspeak/node_modules/line-input-stream/lib/line-input-stream.js:36:9)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)

npm ERR! Linux 3.13.0-92-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "start"
npm ERR! node v6.10.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! steam-ts@1.3.3 start: `node lib/launch.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the steam-ts@1.3.3 start script 'node lib/launch.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the steam-ts package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node lib/launch.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs steam-ts
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls steam-ts
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/steam-ts/npm-debug.log`

**Comment by starwolfy** [comment]
I will attempt to resolve this issue in a couple of days, I will get back to this in around a week.

**Comment by TMFuzz** [comment]
Would a perl script interfere with npm scripts.

**Comment by starwolfy** [comment]
This is a bug in my code, has nothing to do with your scripts. I am currently a bit busy but I will fix this as soon as I can.

**Comment by ZeroTaps** [comment]
I'm getting this issue to Only from today tho

**Comment by starwolfy** [comment]
Fixed as of 9b3a358f30364bd7e111a32e12c921e73daebf43 .

Also sorry for the late fix, closing this issue now. If it still occurs after having updated the project please do reply here again.

**Comment by Forge-Media** [comment]
Sorry for bringing this back up. Having to use PM2 to keep the bot alive. Its crashing 'randomly'.

Seeing the same error as above, even after the fix #9b3a358

```
Error: Callback was already called.
    at /home/ts3bot/steam-ts/node_modules/async/dist/async.js:837:34
    at Object.cb (/home/ts3bot/steam-ts/lib/queryusers.js:49:29)
    at LineInputStream.<anonymous> (/home/ts3bot/steam-ts/node_modules/node-teamspeak/index.js:170:35)
    at LineInputStream.emit (events.js:182:13)
    at LineInputStream.line (/home/ts3bot/steam-ts/node_modules/line-input-stream/lib/line-input-stream.js:8:8)
    at Array.forEach (<anonymous>)
    at Socket.<anonymous> (/home/ts3bot/steam-ts/node_modules/line-input-stream/lib/line-input-stream.js:36:9)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
```

_PM2 lists this error 7 times, and also shows it's restarted the steam-ts bot 7 times, so concluding that this error is causing it to crash._
